test/e2e/tokens_test.go

```diff
@@ -143,7 +145,7 @@ func (suite *TokensTestSuite) TestE2EFungibleTokensAsync() {
 			To:     suite.testState.org2key.Value,
 			Amount: *fftypes.NewFFBigInt(1),
 			From:   suite.testState.org1key.Value,
-			Key:    suite.testState.org2key.Value,
```

Comment:
@awrichar Can you take a quick look at this? If we're transferring tokens from `org1` to `org2` using `org2`'s signing key, shouldn't that have failed before?

Reply:
We're using a single blockchain node, with an embedded signer, in the CLI dev environment created for these tests. So the only reason this would have failed is if `org2` wasn't authorized to transfer the tokens... which it was a few lines earlier:
https://github.com/hyperledger/firefly/blob/5e1120d852aadbf75e37efdab8ea0c9b4f8c9766/test/e2e/tokens_test.go#L101-L108

So I think maybe this was _correct_ in the test before, and might just have needed a comment @calbritt

Reply:
Thanks for making the change @calbritt to keep the original test, but change the API endpoint it uses. So it signs with `org2key` as originally, but uses the org2 node to do it - important when using the E2E test in full k8s deployments against a blockchain with per-member key management.